relecov_tools/json_validation.py

```diff
@@ -261,9 +319,19 @@ def create_invalid_metadata(self, invalid_json, metadata, out_folder):
             self.log.error(logtxt)
             raise
         tag = "Sample ID given for sequencing"
-        seq_id_col = [idx for idx, cell in enumerate(ws_sheet[1]) if tag in cell.value]
-        if seq_id_col:
-            id_col = seq_id_col[0]
+        # Check if mandatory colum ($tag) is defined in metadata.
+        try:
+            id_col = next(
+                idx
+                for idx, cell in enumerate(ws_sheet[1])
+                if cell.value is not None and tag in str(cell.value)
+            )
+        except StopIteration:
+            self.log.error(
+                f"Column with tag '{tag}' not found in the second row of the Excel sheet."
+            )
+            stderr.print(f"[red] Column with tag '{tag}' not found. Cannot continue.")
+            sys.exit(1)
```

Comment:
Same here, I'd raise the error to ease further automation of the process
```suggestion
            raise
```